shared/chat/inbox/row/big-team-channel/index.js

```diff
@@ -25,13 +25,21 @@ type Props = {
   onSelectConversation: () => void,
 }
 
+type State = {
+  isHovered: boolean,
+}
+
 const HoverBox = isMobile
   ? Box
   : glamorous(Box)({
       ':hover': {backgroundColor: globalColors.blueGrey2},
     })
 
-class BigTeamChannel extends PureComponent<Props> {
+class BigTeamChannel extends PureComponent<Props, State> {
+  state = {
```

Comment:
if we're doing this we likely dont need Hoverbox anymore?

Reply:
Using `state` for the hover state is still buggy. I'd rather continue using `glamorous` until we/I can work out a better solution. This is just to get the icon hovers working. I'm open to suggestions for making it better.